### PR TITLE
NXP-28608: return async op result as json

### PR DESCRIPTION
**Pull request by nelsonsilva**
PR would've been created by https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-nsilva/138/ 🔵had GH not been degraded 🤷‍♂️

### CODE REVIEW

File: modules/platform/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/adapters/AsyncOperationAdapter.java [hunk 1]
```diff
@@ -229,14 +235,24 @@ public Object result(@PathParam("executionId") String executionId) throws IOExce
...
-            // if output has a "url" key assume it's a redirect url
+            // if output is a map let's return as json
             if (output instanceof Map) {
+                Map map = (Map) output;
+                // if output has a "url" key make it absolute
                 Object url = ((Map<?, ?>) output).get(RESULT_URL_KEY);
                 if (url instanceof String) {
-                    String baseUrl = VirtualHostHelper.getBaseURL(ctx.getRequest());
-                    return redirect(baseUrl + url);
+                    map = new HashMap((Map) output);
```
Comment (efge) [inline modules/platform/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/adapters/AsyncOperationAdapter.java:244]: `map = new HashMap(map)`

**Comment by efge** [comment]
Unit tests would be needed, in `AsyncOperationAdapterTest` probably

**Comment by nelsonsilva** [comment]
> Unit tests would be needed, in `AsyncOperationAdapterTest` probably

Will do 👍 the only ootb action that exposes the blob is the CSV export...

**Comment by nelsonsilva** [comment]
Tests added with a dummy action exposing a blob.

**Pull request merged**